dev/CommonStyles/GridViewItem_themeresources.xaml

```diff
@@ -0,0 +1,6 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+
+    <Style TargetType="GridViewItem" BasedOn="{StaticResource GridViewItemRevealStyle}" />
```

Comment:
I would prefer to conditional xaml it and rename GridViewItemRevealStyle, since it's not reveal in 21h1